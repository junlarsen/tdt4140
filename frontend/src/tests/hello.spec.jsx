import { expect, describe, it } from "vitest";
import { render } from "@testing-library/react";

describe("react application", () => {
  it("can render an element", () => {
    const component = render(<h1>Hello React root</h1>);
    expect(component.asFragment()).toMatchSnapshot();
  });
});
