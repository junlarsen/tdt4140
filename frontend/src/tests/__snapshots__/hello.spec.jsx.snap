// Vitest Snapshot v1

exports[`react application > can render an element 1`] = `
<DocumentFragment>
  <h1>
    Hello React root
  </h1>
</DocumentFragment>
`;
